apps/prairielearn/src/lib/config.ts

```diff
@@ -82,7 +82,16 @@ const ConfigSchema = z.object({
   /** Sets the default user email in development. */
   authEmail: z.string().nullable().default('dev@example.com'),
   authnCookieMaxAgeMilliseconds: z.number().default(30 * 24 * 60 * 60 * 1000),
-  sessionStoreExpireSeconds: z.number().default(86400),
+  /**
+   * How long a session should be kept alive in the session store.
+   */
+  sessionStoreExpireSeconds: z.number().default(30 * 24 * 60 * 60),
+  /**
+   * Specifies a period of time since the last session store expiration
+   * time update after which the expiration time should be updated again.
+   * This value should be smaller than `sessionStoreExpireSeconds`.
```

Comment:
Given the way this is computed, another way to express this is say that it is the threshold for how much below the default expiration time the current expiration becomes before an update is triggered. In other words, if the current expiration is at most one hour less than 30 days, it is left unchanged.

Reply:
Can you tell I struggled with how to name/explain this? 😅

Is it too unclear at the moment?

Reply:
It's not that unclear, but the name doesn't actually represent what the code is doing. In essence, the code is not checking if you updated in the last hour, instead it's checking if it expires in 29 days and 23 hours, and if it's less than that it updates the expiry date. If you'd prefer to keep the current explanation that's totally fine, I'm just suggesting a different way to see it that is closer to what the code is actually doing. This is of course very much nitpicking.

Reply:
I ended up switching around both the implementation and the comments a lot in a8e6e4a. Is this any better? I tried to incorporate your way of thinking about this and I think it made more sense.

Reply:
Looks clearer now, though you now have two comment blocks in config. Did you mean to remove the first one?

Reply:
Thanks, removed in e3556eb. It got hidden above your GitHub comment that my editor showed inline in the file 😂